multimodal-search-tirg/pods/doc.yml

```diff
@@ -0,0 +1,29 @@
+!BinaryPbIndexer
+with:
+  index_filename: doc.gz
+metas:
+  name: docidx
+  workspace: $WORKDIR
+requests:
+  on:
+    IndexRequest:
+      - !Buffer2URI
+        with:
+          base64: True
+      - !ExcludeQL
+        with:
+          fields:
+            - chunks  
+            - buffer
+      - !KVIndexDriver {}
+    SearchRequest:
+      - !KVSearchDriver
+        with:
+          traversal_paths: ['m']
+      - !ExcludeQL
```

Comment:
I think u do not need this `ExcludeQL` because the `docs` in the `KVIndexer` will not have `embeddings` already